### PR TITLE
invers Robot

### PR DESCRIPTION
**Pull request by BilokonYaroslav**

### CODE REVIEW

File: src/inverseRobot.js [hunk 1]
```diff
@@ -8,6 +8,22 @@
...
+    if (ckeck.includes(Object.values(robot)[i])) {
+      return null;
+    };
+
+    ckeck.push(Object.values(robot)[i]);
+  }
+
+  for (let i = Object.keys(robot).length - 1; i >= 0; i--) {
+    inversRobot[Object.values(robot)[i]] = Object.keys(robot)[i];
+  }
```
Comment (VitaliyBodnar) [inline src/inverseRobot.js:24]: let's improve your solution
actually, you can do it with only one loop

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Good job
let's improve your solution

File: src/inverseRobot.js [hunk 2]
```diff
@@ -8,6 +8,37 @@
...
+
+  // for (let i = 0; i < Object.values(robot).length; i++) {
+  //   if (ckeck.includes(Object.values(robot)[i])) {
+  //     return null;
+  //   };
+
+  //   ckeck.push(Object.values(robot)[i]);
+
+  //   inversRobot[Object.values(robot)[i]] = Object.keys(robot)[i];
+  // }
```
Comment (demetrxx) [inline src/inverseRobot.js:22]: Remove comments

File: src/inverseRobot.js [hunk 3]
```diff
@@ -8,6 +8,37 @@
...
+};
 
 module.exports = inverseRobot;
+// const str = {
+//   Kolli: 'name',
+//   123: 'chipVer',
+//   3: 'wheels',
+// };
+
+// console.log(inverseRobot(str));
```
Comment (demetrxx) [inline src/inverseRobot.js:44]: Same

**Review by demetrxx — CHANGES_REQUESTED** [review]
GJ! But let's keep your code clean and remove unnecessary comments

**Review by demetrxx — APPROVED** [review]
Well done 👍 Go ahead!

**Pull request closed without merging**